tiledb/sm/metadata/test/unit_metadata.cc

```diff
@@ -32,6 +32,8 @@
 
 #include <catch.hpp>
 #include "../metadata.h"
+#include "tiledb/common/common-std.h"
```

Comment:
"common.h" includes "common-std.h"

Use "common-std.h" for files in `common` to avoid infinite recursion. Use "common.h" for everything else.